bibliopixel/main/common_flags.py

```diff
@@ -105,3 +105,7 @@ def get_value(name):
         project_flags['maker'] = {'use_numpy': True}
 
     return project.project_to_animation(desc, project_flags)
+
+
+def extend_path(args):
+    gitty.sys_path.extend((args.path + ':.') if args.path else '.')
```

Comment:
Resolve the fully qualified path with os.getcwd()
Unless gitty is doing some magic here then all that does is add wherever you already are to the python path, but means that if you change dirs, then THAT is in your path, not the old folder. That's kinda the point of this... importing via the project file *should* work for files already in your PWD, but it doesn't always and adding the fully qualified path to your python path fixes that.